packages/atlas/src/views/viewer/MarketplaceView/FeaturedNftsSection/FeatureNftModal.tsx

```diff
@@ -33,13 +33,14 @@ export const FeatureNftModal: FC<FeatureNftModalProps> = ({ isOpen, onClose }) =
   } = useForm<{ url: string }>()
   const [isInputValidating, setIsInputValidating] = useState(false)
 
-  const handleClose = () => {
-    onClose()
-    setVideoId('')
-    reset({ url: '' })
-  }
   const [videoId, setVideoId] = useState('')
+  const abortController = useMemo(() => new AbortController(), [])
```

Comment:
If we want to memo this value only once, maybe we can define it above the fn declaration.

Besides that, all good.